### PR TITLE
chore: added new icons

### LINKED ISSUE #529: [Enhancement] Add missing general icons
**Is your enhancement request related to a problem? Please describe.**
We have some general icons that are missing from our icon-catalogue in Kirby. We wish that those get added.

The missing icons are attached here:
[missing icons.zip](https://github.com/kirbydesign/designsystem/files/3545342/missing.icons.zip)

### PR DESCRIPTION
**Pull request by lydemann**
Closes #529.

Added missing icons.

### CODE REVIEW

File: src/kirby/components/icon/kirby-icon-settings.ts [hunk 1]
```diff
@@ -46,5 +46,20 @@ export const kirbyIconSettings: IconSettings = {
     { name: 'verify', svg: 'assets/kirby/icons/svg/verify.svg', unicode: 0xe927 },
     { name: 'warning', svg: 'assets/kirby/icons/svg/warning.svg', unicode: 0xe928 },
     { name: 'write-message', svg: 'assets/kirby/icons/svg/write-message.svg', unicode: 0xe929 },
+    { name: 'accounts-outline', svg: 'assets/kirby/icons/svg/accounts-outline.svg', unicode: 0xe900 },
```
Comment (jonasjensen-brolstark) [inline src/kirby/components/icon/kirby-icon-settings.ts]: I spot some duplicates entries here. `accounts-outline` is defined on line 7. 

Please remove duplicates.

Comment (lydemann) [reply src/kirby/components/icon/kirby-icon-settings.ts]: Nice spotted :)

File: src/kirby/components/icon/kirby-icon-settings.ts [hunk 2]
```diff
@@ -46,5 +46,19 @@ export const kirbyIconSettings: IconSettings = {
     { name: 'verify', svg: 'assets/kirby/icons/svg/verify.svg', unicode: 0xe927 },
     { name: 'warning', svg: 'assets/kirby/icons/svg/warning.svg', unicode: 0xe928 },
     { name: 'write-message', svg: 'assets/kirby/icons/svg/write-message.svg', unicode: 0xe929 },
+    { name: 'accounts', svg: 'assets/kirby/icons/svg/accounts.svg', unicode: 0xe901 },
```
Comment (lydemann) [inline src/kirby/components/icon/kirby-icon-settings.ts]:
```suggestion
```

File: src/kirby/components/icon/kirby-icon-settings.ts [hunk 3]
```diff
@@ -46,5 +41,17 @@ export const kirbyIconSettings: IconSettings = {
     { name: 'verify', svg: 'assets/kirby/icons/svg/verify.svg', unicode: 0xe927 },
     { name: 'warning', svg: 'assets/kirby/icons/svg/warning.svg', unicode: 0xe928 },
     { name: 'write-message', svg: 'assets/kirby/icons/svg/write-message.svg', unicode: 0xe929 },
+    { name: 'backspace', svg: 'assets/kirby/icons/svg/backspace.svg', unicode: 0xee954 },
```
Comment (jonasjensen-brolstark) [inline src/kirby/components/icon/kirby-icon-settings.ts]: I have one really annoting request. Could you sort the objects alphabetic after the `name`?
It was before 😛

Comment (lydemann) [reply src/kirby/components/icon/kirby-icon-settings.ts]: Well, this is not used anymore, as native is going out. Is it worth it ;)

Comment (jonasjensen-brolstark) [reply src/kirby/components/icon/kirby-icon-settings.ts]: nope not worth it. 😄

**Pull request merged**